jmp-service-rest/src/main/java/com/epam/learning/backendservices/rest/exeption/RestServiceAdvice.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.learning.backendservices.rest.exeption;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ControllerAdvice
+public class RestServiceAdvice {
+
+    @ResponseBody
+    @ExceptionHandler(UserNotFoundException.class)
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    String userNotFoundHandler(UserNotFoundException ex) {
+        return ex.getMessage();
+    }
+
+    @ResponseBody
+    @ExceptionHandler(SubscriptionNotFoundException.class)
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    String subscriptionNotFoundHandler(SubscriptionNotFoundException ex) {
+        return ex.getMessage();
```

Comment:
It is better to send back json payload with error details. Optional: make json representation of an error.